gcovr/formats/html/write.py

```diff
@@ -183,8 +192,9 @@ def highlighter_for_file(self, filename):
 
 @Lazy
 def get_formatter(options):
+    highlight_style = templates(options).get_template("%s.pygments" % get_theme_color(options.html_theme)).render()
```

Comment:
Use f-strings. I think it's better to name the file `pygments.<colour>` to group the files in the file manager.
```suggestion
    highlight_style = templates(options).get_template(f"pygments.{get_theme_color(options.html_theme)}").render()
```